tools/Azure.Mcp.Tools.Redis/src/Commands/ResourceListCommand.cs

```diff
@@ -1,28 +1,28 @@
-// Copyright (c) Microsoft Corporation.
+﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
 using Azure.Mcp.Core.Commands;
 using Azure.Mcp.Core.Commands.Subscription;
-using Azure.Mcp.Tools.Redis.Models.CacheForRedis;
-using Azure.Mcp.Tools.Redis.Options.CacheForRedis;
+using Azure.Mcp.Tools.Redis.Models;
+using Azure.Mcp.Tools.Redis.Options;
 using Azure.Mcp.Tools.Redis.Services;
 using Microsoft.Extensions.Logging;
 
-namespace Azure.Mcp.Tools.Redis.Commands.CacheForRedis;
+namespace Azure.Mcp.Tools.Redis.Commands;
 
 /// <summary>
-/// Lists Azure Cache for Redis resources (Basic, Standard, and Premium tier caches) in the specified subscription.
+/// Lists Redis resources in a subscription. Returns Redis resource details for Azure Cache for Redis (Basic, Standard, and Premium tier), Azure Managed Redis, and Azure Redis Enterprise (`Balanced`, `MemoryOptimized`, `FlashOptimized`, `ComputeOptimized`, `Enterprise`, `EnterpriseFlash`).
```

Comment:
```suggestion
/// Lists Redis resources in a subscription. Returns details for all Azure Managed Redis, Azure Cache for Redis, and Azure Redis Enterprise resources.
```
(listing the tiers was only necessary when different list commands returned different tiers)